com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboard.cs

```diff
@@ -180,9 +181,28 @@ public string Text
         [SerializeField, Tooltip("References the @ bottom panel.")]
         public GameObject emailBottomKeysSection = null;
 
+        /// <summary>
+        /// Used for changing the color of the icon to indicate if recording is active.
+        /// </summary>
+        [SerializeField, Tooltip("Used for changing the color of the icon to indicate if recording is active.")]
+        public Image dictationRecordIcon = null;
```

Comment:
can you make `dictationRecordIcon`, `urlBottomKeysSection `, `defaultBottomKeysSection`,  `alphaKeysSection`, `symbolKeysSection`, `defaultBottomKeysSection`, `urlBottomKeysSection` and `emailBottomKeysSection` a property like the other public keyboard settings, so we remain consistent.  Sorry I missed this before...



for example:

 [SerializeField, Tooltip("Used for changing the color of the icon to indicate if recording is active.")]
 public Image DictationRecordIcon = { get; set; };
 
 
 Note changing this will break the prefab, so you'll need to fix that...sorry